t/t2350-resource-list.t

```diff
@@ -168,7 +168,7 @@ test_expect_success 'flux resource lists expected queues in states (single)' '
 test_expect_success 'cleanup jobs' '
 	flux job cancel $(cat job1A.id) $(cat job1B.id)
 '
-test_expect_success 'configure queues and resource split amongst queues' '
```

Comment:
commit message typo: "t23t0-resource-list" :arrow_right: "t2350-resource-list.t"

Reply:
Oops that was supposed to be part of a review, sorry.